library/silverpeas

```diff
@@ -2,8 +2,8 @@
 Maintainers: Miguel Moquillon <miguel.moquillon@silverpeas.org> (@mmoqui)
 GitRepo: https://github.com/Silverpeas/docker-silverpeas-prod.git
 
-Tags: 6.2.1, latest
-GitCommit: e97e55f0a21aae880a0054ef764cea0cb37e1e05
+Tags: 6.2.2, latest
+GitCommit: c6b68c650da8a95d9b258b4fe1f930fa9be737a6
```

Comment:
Yeah, we added a test in https://github.com/docker-library/official-images/pull/11071 that ensures commits exist in the upstream. The fix is just telling it where to fetch the commits from. Adding a `GitFetch: ref/heads/6.2.x` line for the respective branch on each block should work.

Like this:
```suggestion
GitCommit: c6b68c650da8a95d9b258b4fe1f930fa9be737a6
GitFetch: refs/heads/6.2.x
```

Reply:
Ok, thanks